compiler-plugin/src/main/java/io/ballerina/stdlib/sql/compiler/Constants.java

```diff
@@ -96,6 +96,15 @@ public static class OutParameter {
         public static final String XML = "XMLOutParameter";
     }
 
-    public static final String UNNECESSARY_CHARS_REGEX = "\"|\\n";
+    /**
+     * Constants for regex validation for Ballerina time module record types.
+     */
+    public static class TimeRecordTypesRegex {
+        public static final String TIME_OF_DAY = "ballerina/time:[0-9].[0-9].[0-9]:TimeOfDay";
```

Comment:
Lets just use startsWith("ballerina/time"); and endsWith(<relevantType>)

Reply:
Fixed in [61c4e92](https://github.com/ballerina-platform/module-ballerina-sql/pull/520/commits/61c4e92da35a653eb96a35ccc9f7a7de7421566d)